src/App.tsx

```diff
@@ -1,29 +1,29 @@
-import { Component } from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 
-interface State {
-  movies: Movie[];
-}
+export const App: React.FC<{}> = () => {
+  const [movies, setMovies] = useState<Movie[]>([]);
 
-export class App extends Component<{}, State> {
-  state: State = {
-    movies: [],
-  };
+  const addMovie = (movie: Movie): void => {
+    const foundMovie = movies.every(({ imdbID }) => movie.imdbID !== imdbID);
 
-  render() {
-    const { movies } = this.state;
+    if (foundMovie) {
```

Comment:
И если 
```suggestion
    if (!movieExist) {
```
Если фильма в списке нет - добавляем его